aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -28,15 +36,43 @@ class AztecToolbar : FrameLayout {
         initView()
     }
 
+    override fun onSaveInstanceState(): Parcelable {
+        val bundle = Bundle()
+        bundle.putParcelable("superState", super.onSaveInstanceState())
+        bundle.putBoolean("isUrlDialogVisible", if (addLinkDialog != null) addLinkDialog!!.isShowing else false)
```

Comment:
The dialog visibility is saved, but the link text and URL isn't when you rotate the screen.